src/stan/mcmc/hmc/nuts/base_nuts.hpp

```diff
@@ -152,14 +152,16 @@ namespace stan {
         names.push_back("stepsize__");
         names.push_back("treedepth__");
         names.push_back("n_leapfrog__");
-        names.push_back("n_divergent__");
+        names.push_back("divergent__");
+        names.push_back("E__");
```

Comment:
It’s the value of the Hamiltonian at the sample (included the momentum
which gets thrown away), or energy.  I would prefer we stick with E as it
is concise and matches the notation being used in my papers.  

On Mar 11, 2016, at 7:14 PM, Daniel Lee <notifications@github.com> wrote:

> In src/stan/mcmc/hmc/nuts/base_nuts.hpp:
> 
> > @@ -152,14 +152,16 @@ namespace stan {
> >          names.push_back("stepsize__");
> >          names.push_back("treedepth__");
> >          names.push_back("n_leapfrog__");
> > -        names.push_back("n_divergent__");
> > +        names.push_back("divergent__");
> > +        names.push_back("E__");
> @betanalpha, I got confirmation from @jgabry. It'll break existing things, but not hard to change.
> 
> What's E__? Could we rename that to something descriptive?
> 
> —
> Reply to this email directly or view it on GitHub.
>